pkgs/games/freeciv/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchurl, zlib, bzip2, pkgconfig, curl, lzma, gettext
 , sdlClient ? true, SDL, SDL_mixer, SDL_image, SDL_ttf, SDL_gfx, freetype, fluidsynth
 , gtkClient ? false, gtk2
-, server ? true, readline }:
+, server ? true, enable_sqlite ? true, readline, sqlite }:
```

Comment:
This needs renaming because of code style (we use camelBack case for flags, e.g. `enableSqlite`). I can do it myself if you are okay with `enableSqlite` -- are you?